lua/nvim-tree/lib.lua

```diff
@@ -60,7 +62,7 @@ function M.get_node_at_cursor()
     if TreeExplorer.cwd == "/" then
       line = line + 1
     end
-    return get_node_at_line(line)(TreeExplorer.nodes)
+    return M.get_nodes_by_line(TreeExplorer.nodes, view.View.hide_root_folder and 1 or 2)[line]
```

Comment:
This will be a little slower, as get_nodes_by_line iterates all nodes, rather than returning early as `get_node_at_line` did.